src/Nancy.Swagger/Nancy.Swagger.csproj

```diff
@@ -14,6 +14,7 @@
     <PackageProjectUrl>https://github.com/yahehe/Nancy.Swagger</PackageProjectUrl>
     <PackageLicenseUrl>https://raw.githubusercontent.com/yahehe/Nancy.Swagger/master/LICENSE</PackageLicenseUrl>
     <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netstandard1.6' ">$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
+    <NetStandardImplicitPackageVersion Condition=" '$(TargetFramework)' == 'netstandard1.6' ">1.6.0</NetStandardImplicitPackageVersion>
```

Comment:
Was this the issue? if so how does it resolve the issue with Swagger.ObjectModel?

Reply:
The problem was with Nancy.Swagger.Annotations. 
It was weird - this line was in Swagger.ObjectModel and Nancy.Swagger.Annotations (because I added it before to resolve some other issue), but apparently I didn't add it to Nancy.Swagger. 
When the solution was built, MSBuild would use the same version for all three projects. But when each project was packaged separately, two would be built with 1.6.0 and the other with 1.6.1, which would cause the packing to fail. 

Build Swagger.ObjectModel (NetStandard 1.6.0) -> 
then build Nancy.Swagger (NetStandard 1.6.1 and this is fine) -> 
then build Nancy.Swagger.Annotations (NetStandard 1.6.0, but a 1.6.0 project cannot be built using a 1.6.1 built project)